### DISCUSSION TITLE
Carousel Anything

### DISCUSSION THREAD

**Discussion by mikeoverton** [elementor/elementor#23990] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Being able to make a non "dynamic" carousel. Make custom carousels with varying layouts, a good example of this is apple.com

### Describe the solution you'd like

Would be great if the loop carousel used nested elements instead of a loop. Would mean you can make a slider with any content.

### Describe alternatives you've considered

Ive tried using custom code to turn the grid / flex box to emulate a slider but it lacks features like snapping, Navigation etc. 

Ive tried 3rd party add-ons but they are fixed to certain layouts rather than being able to use nested elements

**Comment by loucasl** [comment]
This already exists as the nested carousel (or in the editor it's just called Carousel). You have to enable the container and nested elements features in the Elementor settings first though

**Reply by mikeoverton** [reply]
Awesome
I cant believe I didnt see this
